public/colouring-images/The Point.svg

```diff
@@ -0,0 +1 @@
le="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3512.65,2758.12l0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3544.44,2758.12l-0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3576.23,2758.12l0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3608.02,2758.12l-0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3639.8,2758.12l0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3671.59,2758.12l-0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3703.38,2758.12l0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3735.17,2758.12l-0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3766.96,2758.12l0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3798.75,2758.12l-0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3830.53,2758.12l0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3862.32,2758.12l-0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><rect x="3891.77" y="2644.37" width="96.483" height="113.758" style="fill:#fff;stroke:#231f20;stroke-width:8.67px;"/><path d="M3921.21,2758.12l-0.001,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3953,2758.12l0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><path d="M3984.8,2758.12l-0,-113.758" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/><g><rect x="2460.19" y="714.092" width="210.946" height="284.175" style="fill:#fff;stroke:#231f20;stroke-width:8.67px;stroke-linecap:square;stroke-miterlimit:1.5;"/><rect x="2502.05" y="752.346" width="127.221" height="203.196" style="fill:#fff;stroke:#231f20;stroke-width:8.67px;"/><path d="M2565.66,753.375l0,202.163" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.58px;"/><path d="M2631.23,857.213l-129.183,-0" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/></g><g><rect x="3421.07" y="714.408" width="210.946" height="284.175" style="fill:#fff;stroke:#231f20;stroke-width:8.67px;stroke-linecap:square;stroke-miterlimit:1.5;"/><rect x="3462.93" y="752.663" width="127.221" height="203.196" style="fill:#fff;stroke:#231f20;stroke-width:8.67px;"/><path d="M3526.54,753.692l-0,202.162" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.58px;"/><path d="M3592.11,857.529l-129.184,0" style="fill:none;fill-rule:nonzero;stroke:#231f20;stroke-width:8.67px;"/></g><rect x="1356.53" y="2212.35" width="318.204" height="40.771" style="fill:#fff;stroke:#231f20;stroke-width:8.67px;"/></svg>
```

Comment:
Do we want the file name to have a space in it?

Reply:
The plan was so that you can just add a file here and it knows its own label, without having to add any extra data.